main/utils.py

```diff
@@ -144,3 +145,22 @@ def get_month_dates_for_previous_year() -> list[tuple[date, date]]:
 
     dates.reverse()
     return dates
+
+
+def order_queryset_by_property(  # type: ignore[explicit-any]
+    queryset: QuerySet[Any], property: str, is_descending: bool
+) -> QuerySet[Any]:
```

Comment:
Would using `Model` instead of `Any` work? Django typing is odd, but hopefully using `Model` would account for any `Model` subclass, like `Funding` or `Project`. Or maybe not...